pyhyp/pyHyp.py

```diff
@@ -405,8 +384,19 @@ def __init__(self, comm=None, options=None, debug=False):
         if options is None:
             raise Error("The options = keyword argument is *NOT* optional. " "It must always be provided")
 
+        # Deprecated options
+        deprecated_options = self._getDeprecatedOptions()
```

Comment:
Thanks. Yes, there are some plugins available for `flake8` (such as [pep8-naming](https://github.com/PyCQA/pep8-naming)). Ideally we should enforce this, but this might take some time to enforce since there will be many instances that need to be changed (and some disabled). Will open an issue to discuss (see https://github.com/mdolab/.github/issues/66)

Reply:
Thanks, sounds good.